### DISCUSSION TITLE
subscription/

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#172] [Comments]
# subscription/

订阅博客

https://hutusi.com/subscription/

**Comment by hi-nikola** [comment]
这个订阅功能很厉害  请问如何实现呢？

**Reply by hutusi** [reply]
我这里提供的是三种订阅功能，请问你说的是哪一个呢？
第一个是RSS，第二个是微信公众号订阅，第三个是用mailchimp提供的邮件订阅。

**Reply by hi-nikola** [reply]
感谢 ！ mailchimp刚刚看了一下 ，准备试一试！

**Comment by Yuxin61** [comment]
rss订阅好像不能阅读全文，我用的reeder

**Reply by hutusi** [reply]
目前RSS没有输出全文，输出的是摘要，主要考虑到文章会被修改，这样跳转到原文查看可以更准确。

**Comment by chaorders1** [comment]
请问还可以通过邮箱订阅吗？ subscribe灰色了

**Reply by hutusi** [reply]
谢谢提醒，我是用mailchimp来做邮箱订阅的，刚发现因为长时间未登录，被删除账号了，回头我再处理一下。
